src/ol/events/event.js

```diff
@@ -47,7 +47,7 @@ ol.events.Event.prototype.stopPropagation = function() {
 /**
  * @param {Event|ol.events.Event} evt Event
  */
-ol.events.Event.stopPropagation = function(evt) {
+ol.events.Event.prototype.stopPropagation = function(evt) {
```

Comment:
This looks like a mistake to me.  There are still a number of places in the library that use the `ol.events.Event.stopPropagation` function.  With this change, that will be `undefined`.  I hope this causes at least a few test failures.

Not sure why you'd want it to be a prototype member.  A name like `ol.events.stopPropagation` could make sense, but it doesn't need to be accessible from `ol.events.Event` instances.